packages/features/form-builder/Components.tsx

```diff
@@ -0,0 +1,385 @@
+                      addOnClassname={classNames(
+                        "border-gray-300 border block border-l-0 disabled:bg-gray-200 disabled:hover:cursor-not-allowed bg-transparent disabled:text-gray-500 dark:border-darkgray-300 "
+                      )}
+                      placeholder={placeholder}
+                      label={<></>}
+                      required
+                      addOnSuffix={
+                        !readOnly ? (
+                          <Tooltip content="Remove email">
+                            <button
+                              className="m-1 disabled:hover:cursor-not-allowed"
+                              type="button"
+                              onClick={() => {
+                                value.splice(index, 1);
+                                setValue(value);
+                              }}>
+                              <FiX className="text-gray-600" />
+                            </button>
+                          </Tooltip>
+                        ) : null
+                      }
+                    />
+                  </li>
+                ))}
+              </ul>
+              {!readOnly && (
+                <Button
+                  type="button"
+                  color="minimal"
+                  StartIcon={FiUserPlus}
+                  className="my-2.5"
+                  onClick={() => {
+                    value.push("");
+                    setValue(value);
+                  }}>
+                  {t("add_another")}
+                </Button>
+              )}
+            </div>
+          ) : (
+            <></>
+          )}
+
+          {!value.length && !readOnly && (
+            <Button
+              color="minimal"
+              variant="button"
+              StartIcon={FiUserPlus}
+              onClick={() => {
+                value.push("");
+                setValue(value);
+              }}
+              className="mr-auto">
+              {label}
+            </Button>
+          )}
+        </>
+      );
+    },
+  },
+  multiselect: {
+    propsType: "multiselect",
+    factory: (props) => {
+      const newProps = {
+        ...props,
+        listValues: props.options.map((o) => ({ title: o.label, value: o.value })),
+      };
+      return <Widgets.MultiSelectWidget {...newProps} />;
+    },
+  },
+  select: {
+    propsType: "select",
+    factory: (props) => {
+      const newProps = {
+        ...props,
+        listValues: props.options.map((o) => ({ title: o.label, value: o.value })),
+      };
+      return <Widgets.SelectWidget {...newProps} />;
+    },
+  },
+  checkbox: {
+    propsType: "multiselect",
+    factory: ({ options, readOnly, setValue, value }) => {
+      value = value || [];
+      return (
+        <div>
+          {options.map((option, i) => {
+            return (
+              <label key={i} className="block">
+                <input
```

Comment:
can we use our `<Checkbox/>` component here? Our checkboxes should have rounded corners

Reply:
Tried using <Checkbox> and it mandates a description field right now, which this component doesn't need because it is going to show a list of checkboxes. Need to think about making changes in the component. 
I am adding it to followup list in the PR and will pick it up. Don't want to do changes to more files in this PR, as it's already huge.

Reply:
Sounds good, maybe you can make rounded corners to the checkboxes for now. That's why I saw that we don't reuse the component here :)

Reply:
Made it rounded.